src/users/04-moderation.md

```diff
@@ -31,3 +31,229 @@ All moderation actions are taken on the context menu of posts or comments. Click
 | Purge user         | Completely delete the user, including all posts and uploaded media. Use with caution.                                                       | Admin            |
 | Purge post/comment | Completely delete the post, including attached media.                                                                                       | Admin            |
 | Appoint as admin   | Gives the user administrator status                                                                                                         | Admin            |
+
+### Image Moderation
+
+> Attribution: This documentation was originally posted by Michael Altfield [here](https://tech.michaelaltfield.net/2024/03/04/lemmy-fediverse-gdpr/)
+
+Currently there is no WUI for deleting images in Lemmy. And deleting accounts does not delete the accounts' images. For more information, please see:
+
+-   [lemmy #4433: Deleted Account should delete uploaded media
+    (pictures) too](https://github.com/LemmyNet/lemmy/issues/4433)
+-   [lemmy #4441: Users unable to delete their images (pictrs
+    API)](https://github.com/LemmyNet/lemmy/issues/4441)
+-   [lemmy #4445: Create an interface for local users to view and remove
+    images](https://github.com/LemmyNet/lemmy/issues/4445)
+-   [lemmy-ui #2359: Allow users to delete images they
+    uploaded](https://github.com/LemmyNet/lemmy-ui/issues/2359)
+-   [lemmy-ui #2360: Allow admins to view & delete uploaded
+    images](https://github.com/LemmyNet/lemmy-ui/issues/2360)
```

Comment:
This issue list will very quickly get outdated as issues get closed, and no one will remember to update it. In fact some are already closed.

Reply:
> no one will remember to update it

I recommend checking your docs every release and updating them. Docs are extremely important.